### DISCUSSION TITLE
Step 2 of Quick Start for Users

### DISCUSSION THREAD

**Discussion by PellTheTabber1** [NOAA-EMC/WW3#1099] [Q&A] [answered]
Step 2: Download and expand large/binary/netcdf files 
cd WW3

sh model/bin/ww3_from_ftp.sh

When the above script runs it errors out on cd tempor/ww3ftp, it says no such directory on the server side (not my side).  I stupidly created the directory  on my side but something is wrong with the above process beyond my control.  Is there a way I can accomplish step 2 using another method to get the netcdf files expanded?  I'm new and only just installed everything.  The install went well so far.

**Comment by PellTheTabber1** [comment]
Can I download the file ww3_from_ftp.v6.07.tar.gz from another site and untar it myself outside the script?  Should I modify the script to some directory that DOES work on the server side?  It's a server side error that fails to find the file due to the directory tempor/ww3ftp no existing.  Has the server been reorganized?

**Reply by MatthewMasarik-NOAA** [reply]
Hi @PellTheTabber1, I'm sorry to hear the ftp download isn't working.  Let me look into it and I'll get back to you today momentarily.

**Reply by MatthewMasarik-NOAA** [reply, marked as answer]
@PellTheTabber1 - unfortunately that version has been removed from the ftp site.  We are just allowed space to keep a few of the most recent versions, currently the two version we have can be seen here:

![image](https://github.com/NOAA-EMC/WW3/assets/86749872/ea36732b-27c6-4877-b196-a1031139f9ca)

The script is callable from the root directory as, `./model/bin/ww3_from_ftp.sh`, and the current `develop` will automatically grab `v7.14.1`.

I would suggest:  1) update your version of WW3, 2) then use the ftp call as shown.

**Reply by PellTheTabber1** [reply]
So I guess I will be building the developer version and using it as the user version?  I will proceed as you suggested and abandon version 6.  Thank you again.

**Reply by MatthewMasarik-NOAA** [reply]
I'm glad to hear that will work @PellTheTabber1!  Thanks for following up.  Cheers

**Comment by PellTheTabber1** [comment]
I guess I need to use the developer script, the User's script no longer works.

wget --no-check-certificate https://ftp.emc.ncep.noaa.gov/static_files/public/WW3/ww3_from_ftp.${ww3ver}.tar.gz
tar -xvzf ww3_from_ftp.${ww3ver}.tar.gz

**Comment by PellTheTabber1** [comment]
Got it! Thank you Matthew, my browser is acting funny so I had to post this thank you as an answer, sorry.
